framework/src/Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic/Themes/Basic/Layouts/Application.cshtml

```diff
@@ -25,17 +25,11 @@
 
         pageTitle = PageLayout.Content.Title + pageTitle;
     }
-
-    var pageLang = ViewBag.Lang == null ? "" : $" lang={ViewBag.Lang}";
-    if (PageLayout.Content.Lang != null)
-    {
-        pageLang = $" lang={PageLayout.Content.Lang}";
-    }
 }
 
 <!DOCTYPE html>
 
-<@($"html{pageLang}")>
+<html lang="@CultureInfo.CurrentCulture.Name">
```

Comment:
It should be **CurrentUICulture**. While for the most time they are identical, there may be rare cases when they can differ. **CurrentCulture** in .NET controls number, date, currency formatting. While **CurrentUICulture** represents language of the text, which is same function as `<html lang="">` attribute.

Possible use case: User may like to stick with default English language text, but may also prefer his native formatting to better understand dates and numbers which is allowed by string.Format();`, but document should still me marked "en".

Reply:
I see that there is more priority in using CurrentCulture in the framework.

Reply:
I don't understand. In the framework CurrentUICulture is used to determine which resources to use. There is no ambiguity, CurrentCulture is not meant to determine document language and I don't see it being used for anything bar being assigned in the framework or working as fallback if CurrentUICulture is null (which I believe can't happen).

Reply:
@hikalkan What do you think?